### ISSUE TITLE
Zarr.jl

### ISSUE THREAD

**Issue by joshmoore** [constantinpape/zarr_implementations#42]
See: https://github.com/meggart/Zarr.jl

@meggart, would there be interest in having Zarr.jl tested against other implementations?

----

Steps:
- [ ] (Primarily) add a driver under [generate_data/](https://github.com/zarr-developers/zarr_implementations/tree/master/generate_data) that minimally generates but optionally also reads zarr data (see [jzarr](https://github.com/zarr-developers/zarr_implementations/blob/master/generate_data/jzarr/src/main/java/zarr_implementations/jzarr/App.java#L83) example)
- [ ] if implementing reading, add a [read_with_julia](https://github.com/zarr-developers/zarr_implementations/blob/master/test/test_read_all.py#L79) method
- [ ] update [environment](https://github.com/zarr-developers/zarr_implementations/blob/master/environment.yml) with Julia requirements
- [ ] add [Makefile](https://github.com/zarr-developers/zarr_implementations/blob/master/Makefile) targets

**Comment by meggart** [comment]
Yes, of course. I always wanted to do this but did not get to it yet. I have a few unit tests that compare against zarr-python (https://github.com/meggart/Zarr.jl/blob/master/test/python.jl), but something more systematic would be great.

**Comment by joshmoore** [comment]
A perhaps interesting test to include: https://github.com/meggart/Zarr.jl/issues/78#issuecomment-1006112368

**Comment by mkitti** [comment]
I'll volunteer to help on this (or perhaps I have been volunteered) although I think @meggart has the lead here. What needs to be done?

Maybe the first step is to address meggart/Zarr.jl#25. I'm already a member of https://github.com/JuliaIO, so I'm biased in that direction.

**Comment by joshmoore** [comment]
Thanks for the offer, @mkitti! I've updated the description with the steps that I know of. From the perspective of this repo, happy to point to whatever org even temporarily and we can always update when https://github.com/meggart/Zarr.jl/issues/25 is settled.

**Comment by meggart** [comment]
I agree with @joshmoore . Thanks a lot for offering your help here and we can start writing these coparison tests for Julia independent on where the package lives. Thanks also for updating the todo list @mkitti let me know what kind of support you would need to write the tests, of course I would be available for reviewing code.

**Comment by grlee77** [comment]
The instructions in the docs proposed in #44 should help. Please provide feedback there if you find things that should be added or clarified.

**Comment by mkitti** [comment]
Zarr.jl is now in JuliaIO.
